cparser/regexParser.js

```diff
@@ -260,8 +184,8 @@ module.exports = that = {
 			// Comments
 			"|(/\\*[^*]*(?:\\*(?!/)[^*]*)*\\*/)|(//.*?$)",
 
-			// Pre-processor
-			"|(#(?:\\\\\\n|.)*)"
+			// Include statements
+			"|(#include.+$)"
```

Comment:
I think we want to include application.h before existing includes, rather than after them... in case those includes use symbols defined in Particle system firmware like `D7`.  Otherwise there is a hard dependency that any library includes must compile on their own, i.e. Particle.h/application.h/Arduino.h must be included in the library.  That would be ok as well.  Another reason you might want application.h on line 1, is if you have added #defines using Particle symbols at the top of your main application file, e.g. `#define MYLED D7`.  application.h could just go to line 1, and the rest could go to before the first statement line.